log_outgoing_requests/tasks.py

```diff
@@ -2,6 +2,13 @@
 from .constants import SaveLogsChoice
 
 
+@shared_task
+def cleanup_requests_logs():
+    from .models import OutgoingRequestsLog
+
+    OutgoingRequestsLog.objects.cleanup_requests_logs()
```

Comment:
```suggestion
    return OutgoingRequestsLog.objects.cleanup_requests_logs()
```
the return value of a celery task is stored as the task result, if a result storage is used. This is then visible in celery-flower, for example.